### DISCUSSION TITLE
Unable to get readings with new Dexcom G7

### DISCUSSION THREAD

**Discussion by Jerr552** [NightscoutFoundation/xDrip#3824] [Bug reports] [answered]
After going through the guide (https://navid200.github.io/xDrip/docs/Dexcom/G7.html) to setup my xDrip app for a new Dexcom G7 (just switched from G6), my status page shows connections every 5 minutes without pulling any readings.  I have installed and uninstalled the Dexcom app to prove that data is being broadcast.  My status screen just shows the following
![image](https://github.com/user-attachments/assets/b760d738-29fe-4c9d-b13a-d86f4edee15b)

**Comment by Navid200** [comment, marked as answer]
If you uninstall Dexcom app and wait long enough it should connect.

This explains what many experience who are not aware of the differences between G6 and G7: https://github.com/NightscoutFoundation/xDrip/discussions/3813  

Alternatively, you can wait 15 minutes to make sure G7 is in rapid reconnect mode meaning transmitting once a minute.
Then, delete all connected devices starting with DXCM or DX01 or DX02.
Then, change the last character of the transmitter ID, apply and change back to the correct value.
Then, go to the status page and wait for a pair request.
After you approve the pair request, you have to pay attention to see if xDrip is scanning or not while still being on the status page.  If it is not, you should tap on restart collector on the classic status page and go back to the Dex status page.  After a second connection, xDrip will have found the correct once-every-5-minute interval and remain properly operational.
